terraform/modules/configuration/configuration.tf

```diff
@@ -67,5 +67,15 @@ eco:
     interval: ${var.eco_snapshot_interval}
     ttl: ${var.eco_snapshot_ttl}
     bucket: ${var.eco_snapshot_bucket}
+EOT
+   telegraf_configuration = <<EOT
+[inputs.prometheus]
+## An array of urls to scrape metrics from.
+urls = ["http://localhost:2381/metrics"]
+metric_version = 2
+[outputs.graphite]
+graphite_tag_support = true
+prefix = "${var.telegraf_graphite_prefix}"
+servers = ["${var.telegraf_graphite_uri}"]
```

Comment:
if we didn't want to pass specific graphite cfgs, we could ask people to provide a whole [outputs] block as a string? how do you feel about that?

Reply:
Yeah that's fair 👍

Reply:
not 100% sure it'll work (e.g. new lines..) but may be worth a try